glib/src/subclass/types.rs

```diff
@@ -305,22 +323,22 @@ unsafe extern "C" fn interface_init<T: ObjectSubclass, A: IsImplementable<T>>(
 ) where
     <A as ObjectType>::GlibClassType: Copy,
 {
-    let iface = &mut *(iface as *mut crate::Interface<A>);
+    let klass = &mut *(iface as *mut crate::Interface<A>);
```

Comment:
OOC, could we use `g_type_class_peek_parent()` here instead of making our own copy?